src/components/list/list.js

```diff
@@ -202,18 +219,18 @@ function mdListItemDirective($mdAria, $mdConstant, $mdUtil, $timeout) {
 
             $scope.mouseActive = false;
             proxy.on('mousedown', function() {
-              $scope.mouseActive = true;
-              $timeout(function(){
-                $scope.mouseActive = false;
-              }, 100);
-            })
-            .on('focus', function() {
-              if ($scope.mouseActive === false) { $element.addClass('md-focused'); }
-              proxy.on('blur', function proxyOnBlur() {
-                $element.removeClass('md-focused');
-                proxy.off('blur', proxyOnBlur);
-              });
-            });
+                  $scope.mouseActive = true;
+                  $timeout(function(){
+                    $scope.mouseActive = false;
+                  }, 100);
+                })
+                .on('focus', function() {
+                  if ($scope.mouseActive === false) { $element.addClass('md-focused'); }
+                  proxy.on('blur', function proxyOnBlur() {
+                    $element.removeClass('md-focused');
+                    proxy.off('blur', proxyOnBlur);
+                  });
+                });
```

Comment:
On Angular Material we have strict rules with indention, please don't reformat existing code.
Indent with 2 Spaces